ipn/ipnlocal/serve.go

```diff
@@ -564,18 +565,43 @@ func (b *LocalBackend) proxyHandlerForBackend(backend string) (http.Handler, err
 // has application/grpc content type header, the connection will be over h2c.
 // Otherwise standard Go http transport will be used.
 type reverseProxy struct {
-	logf     logger.Logf
-	url      *url.URL
+	logf logger.Logf
+	url  *url.URL
+	// set to true if the backend serves https and the connection should be
```

Comment:
For comments not on the same line, use complete sentences beginning with the field name as the subject.

// insecure is whether ....